src/component/LandingPage.jsx

```diff
@@ -0,0 +1,52 @@
+import Header from './Header';
+import Line from './icons/Line';
+import ExhibitionIcon1 from './icons/ExhibitionIcon1';
+import ExhibitionIcon2 from './icons/ExhibitionIcon2';
+import BackIcon from './icons/BackIcon';
+import WAIcon from './icons/WAIcon';
+
+export default function LandingPage() {
+  return (
+    <div>
+      <Header />
+      <div className='overflow-hidden bg-[#3936E5] text-[#FFFFFF]'>
+        <div className='relative mx-auto max-w-[1440px] px-20 py-24'>
+          <p className='font-pretendard text-3xl leading-[40px] font-normal tracking-[-0.025em] text-[#CCD7F2]'>
+            SMWU Programming Club
+          </p>
+          <div className='font-pretendard relative mt-5 grid grid-cols-2 text-9xl leading-none font-bold tracking-[-0.06rem] text-[#FFFFFF]'>
+            <div>
+              <h1>APPS</h1>
```

Comment:
APPS 라는 텍스트에는 스타일 지정이 되어있어서 h1 대신 p를 쓰고 스타일을 적용해보는 건 어떨까요? 

```suggestion
<p className='text-white text-[180px] font-bold leading-[200px]>APPS</p>
```